src/app/public/src/modules/sidebar/sidebar-wrapper.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="stache-sidebar-wrapper" [ngClass]="{'sidebar-closed': sidebarClosed, 'sidebar-open': !sidebarClosed  }">
+  <div class="stache-sidebar-header">
+    <button *ngIf="!sidebarClosed" class="sky-btn sky-btn-link-inline stache-sidebar-button" (click)="closeSidebar()" aria-label="sidebar close">
```

Comment:
Need to set `type="button"` for all `button` elements, otherwise they could (potentially) submit forms on the page.